### PR TITLE
Html docs fixes+adding permissions argument

### PR DESCRIPTION
**Pull request by reutshal**
## Status
Ready

## Description
- Added 'permissions' argument with the following options:
   - per-command - the permissions entry will be diaplayd in every command section. this is default
   - global- the permissions entry will be diaplayd once, in it's own section
   - none- if there are no permissions required to this integration, there will be no such entry
- Added to each command html section - a comment with html example to manually add an image
- Fixed the bug in the arguments descriptions

## Must have
- [ ] Tests
- [ ] Documentation (with link to it)
- [X ] Code Review

### CODE REVIEW

File: Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py [hunk 1]
```diff
@@ -471,6 +491,10 @@ def generate_html_docs(args, yml_data, example_dict, errors):
     # # Setup integration on Demisto
     # docs.extend(generate_setup_section(yml_data))
 
+    #  Permissions
+    if demisto.args().get('permissions') == 'global':
```
Comment (yaakovi) [inline Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py]: don't use `demisto.args()`, use the function argument `args`

File: Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py [hunk 2]
```diff
@@ -370,6 +382,13 @@ def generate_commands_section(yaml_data, example_dict):
     return (COMMANDS_HEADER.format(command_list='\n'.join(command_list)) + '\n'.join(command_sections)), errors
 
 
+def get_permissions_entry():
+    if demisto.args().get('permissions') == 'per-command':
```
Comment (yaakovi) [inline Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py]: don't use `demisto.args()`, pass the variable from main

File: Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py [hunk 3]
```diff
@@ -378,6 +397,7 @@ def generate_single_command_section(index, cmd, example_dict):
         'command_hr': cmd['name'],
         'command': cmd['name'],
         'command_description': cmd.get('description', ' '),
+        'permissions': get_permissions_entry(),
```
Comment (yaakovi) [inline Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py]: pass a boolean here to indicate if the permission section needs to be inserted and then you can simply do:
`'permissions': PERMISSIONS_PER_COMMAND if should_include_permissions else '',`

File: Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py [hunk 4]
```diff
@@ -338,46 +355,37 @@ def generate_section(title, data):
...
-    for conf in yaml_data['configuration']:
-        if conf['display']:
-            section.append('    * __{}__'.format(conf['display']))
-        else:
-            section.append('    * __{}__'.format(conf['name']))
-    section.append('4. Click __Test__ to validate the URLs, token, and connection.')
-
-    return section
+    params_list = [
+        PARAMS_LIST.format(param=conf['display']) if conf['display'] else PARAMS_LIST.format(param=conf['name']) for
```
Comment (yaakovi) [inline Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py]:
```suggestion
        PARAMS_LIST.format(param=conf['display'] if conf.get('display') else conf['name']) for
```

**Comment by reutshal** [comment]
@kirbles19 could you please go over this files:
- CHANGELOG: https://github.com/demisto/content/blob/HTMLDocs-fixes/Scripts/HTMLDocsAutomation/CHANGELOG.md 
- The PERMISSIONS_HEADER global variable (this will be displayed once)
- The PERMISSIONS_PER_COMMAND global variable
- The SETUP_CONFIGURATION global variable,
All in: https://github.com/demisto/content/blob/HTMLDocs-fixes/Scripts/HTMLDocsAutomation/HTMLDocsAutomation.py

Thanks

**Comment by kirbles19** [comment]
@reutshal All done.

**Pull request merged**